### PR TITLE
Switch from a single switch for "clear other items" to one for weapons and armor

### PR DESCRIPTION
**Pull request by bhollis**
This allows for individual control of whether to move aside weapons, armor, or both:
Screenshot 2023-07-17 at 12 48 21 PM

### CODE REVIEW

File: config/i18n.json [hunk 1]
```diff
@@ -652,7 +652,9 @@
     "ClassTypeMissing": "You do not have a {{className}} to create a loadout for",
     "ClearSection": "Remove all",
     "ClearLoadoutParameters": "Remove Loadout Optimizer settings",
-    "ClearSpace": "Move other items away",
+    "ClearSpace": "Move others away",
```
Comment (bhollis) [inline config/i18n.json:655]: This change was mostly to prevent the switch from wrapping to two lines

File: config/i18n.json [hunk 2]
```diff
@@ -652,7 +652,9 @@
...
+    "ClearSpaceWeapons": "Move other weapons away",
+    "ClearSpaceArmor": "Move other armor away",
```
Comment (bhollis) [inline config/i18n.json:657]: These are from the D1 loadout editor, which I didn't want to spend a lot of time on

**Comment by robojumper** [comment]
Can we show the "Move others away" toggle in a category only if the Loadout has any items in a bucket for that category? Since loadout application will only move other items away for buckets where the Loadout has items, it'd maybe be less confusing if ineffective toggles were hidden.

**Comment by bhollis** [comment]
Great idea, I've added that in

**Review by robojumper — APPROVED** [review]

**Pull request merged**